h2/src/main/org/h2/table/Column.java

```diff
@@ -133,8 +146,12 @@ public int hashCode() {
         return table.getId() ^ name.hashCode();
     }
 
+    public boolean isEnumerated() {
+        return enumerators != null && enumerators.length > 0;
```

Comment:
Hey @grandinj as I have things set up now `enumerators` will always be `NULL` for all non-`ENUM` data types. Maybe it would be better to simply check if `dataType == Value.ENUM`? I'm going to make that change now.

With regards to your comment "if we have zero length array, we have other problems" - would you prefer I set things up so that `enumerators` is never `NULL`, even for non-`ENUM` data types?